admin-cli/src/index.ts

```diff
@@ -61,7 +70,41 @@ namespace v3 {
         console.log(`Accepted EmployeeProposal for ${employee}.`);
       }
     }
+
+    // Create/approve employee vacation requests.
+    for (const employee in config.employees) {
+      const employeeInfo = config.employees[employee];
+      for (const vacationRequest of employeeInfo.vacationRequests) {
```

Comment:
Assuming the make the `vacationRequests` field optional:
```suggestion
      for (const vacationRequest of employeeInfo.vacationRequests ?? []) {
```